extensions/ql-vscode/src/data-extensions-editor/predicates.ts

```diff
@@ -82,7 +82,16 @@ export const extensiblePredicateDefinitions: Record<
         kind: row[7] as string,
       },
     }),
-    supportedKinds: ["sql", "xss", "logging"],
+    supportedKinds: [
+      "command-injection",
+      "html-injection",
+      "log-injection",
+      "response-splitting",
+      "path-injection",
+      "sql-injection",
+      "url-redirection",
+      "xpath-injection",
+    ],
```

Comment:
In the long term, we should be generating this list directly from the core libraries themselves.

Reply:
@aeisenberg Makes complete sense and would be amazing to be done automatically.